pkg/apis/dmi/v1/api.proto

```diff
@@ -0,0 +1,324 @@
+syntax = "proto3";
```

Comment:
after the pr(https://github.com/kubeedge/kubeedge/pull/4013) is merged, we can remove this `pkg/apis/dmi` directory and import the package in kubeedge directly

Reply:
only can be merged after v1.12.0 release?